src/rust/ensogl/lib/core/src/gui.rs

```diff
@@ -2,3 +2,4 @@
 
 pub mod component;
 pub mod cursor;
+#[macro_use] pub mod style;
```

Comment:
Please double check if its needed - I never needed to put something like that in sources